src/lib/components/navbar/Navbar.component.js

```diff
@@ -48,6 +55,26 @@ const NavbarMenu = styled.div`
   align-items: center;
 `;
 
+const NavbarTabs = styled.div`
+flex: 1;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const TabItems = styled.div`
+  height: 100%;
+  display: flex;
+  justify-content: center;
+  align-items: center;  
+  margin: 0 1vh;
```

Comment:
please use our predefined padding from `theme.js` to be consistent.
```

import {padding} from "../../style/theme"; 
```

Reply:
okay, didn't see this. I think creating a CONTRIBUTE, may be helpful in the future.